xcdat/regridder/base.py

```diff
@@ -1,68 +1,59 @@
 import abc
-from typing import Any
+from typing import Any, List, Tuple, Union
 
+import numpy as np
 import xarray as xr
 
 import xcdat.bounds  # noqa: F401
 from xcdat._logger import _setup_custom_logger
+from xcdat.axis import CFAxisKey
 
 logger = _setup_custom_logger(__name__)
 
+Coord = Union[np.ndarray, xr.DataArray]
+
+CoordOptionalBnds = Union[Coord, Tuple[Coord, Coord]]
+
 
 def preserve_bounds(
```

Comment:
Do you think this function will be used publicly, or should it be private (`_preserve_bounds()`)?